core/src/main/java/com/homeaway/streamingplatform/extensions/schema/CompatibilityLevel.java

```diff
@@ -0,0 +1,20 @@
+/* Copyright (c) 2018 Expedia Group.
+ * All rights reserved.  http://www.homeaway.com
+
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+
+ *      http://www.apache.org/licenses/LICENSE-2.0
+
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.homeaway.streamingplatform.extensions.schema;
+
+public enum CompatibilityLevel {
+    BACKWARD_TRANSITIVE, FORWARD_TRANSITIVE, FULL_TRANSITIVE
```

Comment:
Can this not reuse this? https://github.com/confluentinc/schema-registry/blob/4.1.2-post/client/src/main/java/io/confluent/kafka/schemaregistry/avro/AvroCompatibilityLevel.java#L19

Reply:
I was confused as to why the SchemaManager reference interface [here](https://homeaway.github.io/stream-registry/architecture/#schema-manager-interface) had fewer compatibility levels.  Until @neoword sees this I will switch it to use the existing one

Reply:
@westeras, yes that one is good.

However, we want ours to be agnostic of others... so that other people can swap in schema registries.  By defining compatibility here, we can plug into the compatibility levels that underlying schema registries provide.